tests/conftest.py

```diff
@@ -990,17 +990,17 @@ def fixture_sample_sheet_context(cg_context: CGConfig, lims_api: LimsAPI) -> CGC
     return cg_context
 
 
-@pytest.fixture
-def bcl_convert_demultiplexed_flow_cell_sample_ids() -> List[str]:
+@pytest.fixture(name="bcl_convert_demultiplexed_flow_cell_sample_internal_ids")
```

Comment:
Is there a reason why you didn't add the scope fo this fixture?

Reply:
I just refactored the name of the fixture. No scope was added prior. Should I add it?